includes/class-newspack-newsletters.php

```diff
@@ -82,6 +82,11 @@ public function __construct() {
 		add_action( 'save_post_' . self::NEWSPACK_NEWSLETTERS_CPT, [ __CLASS__, 'save' ], 10, 3 );
 		add_action( 'admin_enqueue_scripts', [ __CLASS__, 'branding_scripts' ] );
 		add_filter( 'newspack_theme_featured_image_post_types', [ __CLASS__, 'support_featured_image_options' ] );
+
+		/**
+		 * Set the email service provider, which will instantiate the appropriate class
+		 * (Mailchimp, ConstantContact, etc.).
+		 */
```

Comment:
Probably due to a `master` merge, this comment moved one line up - it should be above the line _after_ the next one.